packages/create-vite/template-svelte/README.md

```diff
@@ -16,7 +16,6 @@ Check out [SvelteKit](https://github.com/sveltejs/kit#readme), which is also pow
 
 - It brings its own routing solution which might not be preferable for some users.
 - It is first and foremost a framework that just happens to use Vite under the hood, not a Vite app.
-  `vite dev` and `vite build` wouldn't work in a SvelteKit environment, for example.
```

Comment:
took the liberty to also incorporate this readme change. That line is no longer true, vite dev/build is how you run SvelteKit now.